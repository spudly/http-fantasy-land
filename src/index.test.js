import * as exports from './index.js';
import map from 'ramda/src/map';

test('exports functions', () => {
  const exportTypes = map(value => typeof value, exports);
  expect(exportTypes).toMatchSnapshot();
});
