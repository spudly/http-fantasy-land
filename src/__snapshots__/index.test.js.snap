// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports functions 1`] = `
Object {
  "composeRoutes": "function",
  "connect": "function",
  "del": "function",
  "get": "function",
  "head": "function",
  "listen": "function",
  "options": "function",
  "patch": "function",
  "post": "function",
  "put": "function",
  "sendFile": "function",
  "setHeader": "function",
  "trace": "function",
}
`;
